### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by mahmoud-dev97**
add new navbar branch
change navbar logo
change input and links hover color

### CODE REVIEW

File: frontend/src/app/client/core/components/header/header.component.html [hunk 1]
```diff
@@ -1,8 +1,11 @@
 <nav class="navbar navbar-expand-lg navbar-light">
   <div class="container-fluid">
-    <a routerLink="/" class="navbar-brand fw-bold fs-4 mt-2 col-lg-3">{{
-      title | titlecase
-    }}</a>
+    <div class="col-lg-3">
+      <a routerLink="/" class="navbar-brand mt-2">
+        <img src="../../../../../assets/pngwing.com.png" alt="" />
```
Comment (magdykashef) [inline frontend/src/app/client/core/components/header/header.component.html]: Bind the src to a brand variable with the path 'assets/pngwing.con.png'

Comment (mahmoud-dev97) [reply frontend/src/app/client/core/components/header/header.component.html]: fix done

**Comment by mahmoud-dev97** [comment]
fix src and alt

**Comment by magdykashef** [comment]
Well done Mahmoud

**Review by magdykashef — APPROVED** [review]

**Pull request merged**